### DISCUSSION TITLE
Unable to import web3

### DISCUSSION THREAD

**Discussion by natnaelab** [nuxt/nuxt#17279] [Questions]
```
<script>
import Web3 from "web3";

export default {
  async mounted() {
    console.log(Web3.givenProvider);
  },
};
</script>
```

Output on the console
`util.js:109 Uncaught TypeError: Cannot read properties of undefined (reading 'NODE_DEBUG')`

**Comment by TheAlexLichter** [comment]
Hey! I cannot reproduce the problem in [my CodeSandbox](https://codesandbox.io/s/nuxt-discussion-4951-v783h2). `Web3` seems to be available there (`givenProvider` not due to WS availabilities, but `Web3` is loaded correctly).

**Reply by natnaelab** [reply]
That worked for me as well, however, using Web3 in "methods" causes the issue I mentioned earlier.

**Reply by natnaelab** [reply]
For now, https://github.com/nuxt/framework/discussions/4393 resolved my problem.

**Comment by wobsoriano** [comment]
Published a Nuxt 3 module for this: https://github.com/wobsoriano/nuxt-web3.js

**Reply by andreemic** [reply]
how would you use it outside of nuxt components/composables, in a helper file?
